pnpm-lock.yaml

```diff
@@ -88,15 +88,15 @@ importers:
       classnames: 2.3.1
       copy-to-clipboard: 3.3.1
       js-cookie: 3.0.1
-      next: 12.1.6_37alspmz7orhfskwjuyttisydi
+      next: 12.1.6_apvyc6hsquzbx56wwdowrbgkzm
```

Comment:
I have no idea what happened here. There is, for some reason, a bonus specifier appended. The integrity check of the package is identical, so no underlying changes, so it's weird diff noise.